operator/internal/handlers/internal/gateway/tenant_configmap_test.go

```diff
@@ -80,6 +85,11 @@ func TestGetTenantConfigMapData_ConfigMapExist(t *testing.T) {
 				CookieSecret: "test789",
 			},
 		},
+		"network": {
+			OpenShift: &manifests.TenantOpenShiftSpec{
+				CookieSecret: "test101112",
+			},
+		},
```

Comment:
I don't believe this test case adds more value as it only adds an additional tenant to the stub and expects to find it after calling `GetTenantConfigMapData`. There is nothing `network`-tenant-specific added to the `GetTenantConfigMapData` itself.

Reply:
Sure, I can remove that part